pkg/api/v1beta3/types.go

```diff
@@ -642,7 +642,7 @@ type Container struct {
 	// Deprecated - see SecurityContext.  Optional: Default to false.
 	Privileged bool `json:"privileged,omitempty" description:"whether or not the container is granted privileged status; defaults to false; cannot be updated; deprecated;  See SecurityContext."`
 	// Optional: Policy for pulling images for this container
-	ImagePullPolicy PullPolicy `json:"imagePullPolicy,omitempty" description:"image pull policy; one of PullAlways, PullNever, PullIfNotPresent; defaults to PullAlways if :latest tag is specified, or PullIfNotPresent otherwise; cannot be updated"`
+	ImagePullPolicy PullPolicy `json:"imagePullPolicy,omitempty" description:"image pull policy; one of Always, Never, IfNotPresent; defaults to Always if :latest tag is specified, or IfNotPresent otherwise; cannot be updated"`
```

Comment:
Yes

> On May 22, 2015, at 3:46 PM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/api/v1beta3/types.go:
> 
> > @@ -642,7 +642,7 @@ type Container struct {
> >  	// Deprecated - see SecurityContext.  Optional: Default to false.
> >  	Privileged bool `json:"privileged,omitempty" description:"whether or not the container is granted privileged status; defaults to false; cannot be updated; deprecated;  See SecurityContext."`
> >  	// Optional: Policy for pulling images for this container
> > -	ImagePullPolicy PullPolicy `json:"imagePullPolicy,omitempty" description:"image pull policy; one of PullAlways, PullNever, PullIfNotPresent; defaults to PullAlways if :latest tag is specified, or PullIfNotPresent otherwise; cannot be updated"`
> > +	ImagePullPolicy PullPolicy `json:"imagePullPolicy,omitempty" description:"image pull policy; one of Always, Never, IfNotPresent; defaults to Always if :latest tag is specified, or IfNotPresent otherwise; cannot be updated"`
> Good catch. The same problem exists in v1/types.go. Maybe a separate PR to fix, though?
> 
> —
> Reply to this email directly or view it on GitHub.
>